tests/limiter/limiter.js

```diff
@@ -37,124 +51,620 @@ describe('tests/limiter/limiter.js >', () => {
+      conf.expressLimiterExpire2 = undefined;
+      req = {headers: {}};
+      req.headers.UserName = tu.userName;
+      res = {headers: {}};
+      res.set = function (name, value) { this.headers[name.toLowerCase()] = value };
+    });
+
+    it('ok', (done) => {
+      rateLimit(req, res)
+      .then(() => expectLimited(res))
+      .then(done)
+      .catch(done);
+    });
+
+    it('missing lookup', (done) => {
+      conf.expressLimiterLookup = [];
+      rateLimit(req, res)
+      .then(() => expectNotLimited(res))
+      .then(done)
+      .catch(done);
+    });
+
+    it('empty lookup', (done) => {
+      conf.expressLimiterLookup = [''];
+      rateLimit(req, res)
+      .then(() => expectNotLimited(res))
+      .then(done)
+      .catch(done);
+    });
+
+    it('invalid lookup (undefined)', (done) => {
+      conf.expressLimiterLookup = ['headers.aaa'];
+      rateLimit(req, res)
+      .then(() => expectNotLimited(res))
+      .then(done)
+      .catch(done);
+    });
+
+    it('invalid lookup (error)', (done) => {
+      conf.expressLimiterLookup = ['headers.aaa.bbb'];
+      rateLimit(req, res)
+      .then(() => expectNotLimited(res))
+      .then(done)
+      .catch(done);
+    });
+
+    it('invalid lookup (error, multiple)', (done) => {
+      conf.expressLimiterLookup = ['headers.aaa.bbb', 'headers.UserName'];
+      rateLimit(req, res)
+      .then(() => expectLimited(res))
+      .then(done)
+      .catch(done);
+    });
+
+    it('lookup field undefined', (done) => {
+      conf.expressLimiterLookup = ['headers.UserName'];
+      req.headers.UserName = undefined;
+      rateLimit(req, res)
+      .then(() => expectNotLimited(res))
+      .then(done)
+      .catch(done);
+    });
+
+    it('missing total', (done) => {
+      conf.expressLimiterTotal = undefined;
+      rateLimit(req, res)
+      .then(() => expectNotLimited(res))
+      .then(done)
+      .catch(done);
+    });
+
+    it('missing expire', (done) => {
+      conf.expressLimiterExpire = undefined;
+      rateLimit(req, res)
+      .then(() => expectNotLimited(res))
+      .then(done)
+      .catch(done);
+    });
+
+    it('empty total', (done) => {
+      conf.expressLimiterTotal = '';
+      rateLimit(req, res)
+      .then(() => expectNotLimited(res))
+      .then(done)
+      .catch(done);
+    });
+
+    it('empty expire', (done) => {
+      conf.expressLimiterExpire = '';
+      rateLimit(req, res)
+      .then(() => expectNotLimited(res))
+      .then(done)
+      .catch(done);
+    });
   });
 
-  it('(j) Admin user req#3, no ratelimit headers, ok', (done) => {
-    api.post(path)
-    .set('Authorization', predefinedAdminUserToken)
-    .send({ name: tu.namePrefix + 'Limiter7', timeout: '1m' })
-    .expect(constants.httpStatus.CREATED)
-    .expect((res) => {
-      expect(res.headers).to.not.have.property('x-ratelimit-limit');
-    })
-    .end(done);
+  describe.skip('path/method - fork subprocess >', function() {
```

Comment:
is this .skip intentional?

Reply:
Yeah, this is the only way I could test the path/method config but it takes a long time to execute. I skipped it so it doesn't make the tests take forever.